src/front/config/testnet/swapContract.js

```diff
@@ -15,6 +15,9 @@ export default {
   pancakeswapRouter: '0x9Ac64Cc6e4415144C455BD8E4837Fea55603e5c3', // BSC testnet
   pancakeswapFactory: '0xB7926C0430Afb07AA7DEfDE6DA862aE0Bde767bc',
   sushiswapRouter: '0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506', // Polygon testnet
+  CarthageFactory: '0x5Bb7BAE25728e9e51c25466D2A15FaE97834FD95',
+  CarthageRouter: '0x1a45c9f823F64c9360b0c3df269F824cf404f82b', // Candle
```

Comment:
First letter:
`CarthageRouter` -> `carthageRouter`

The same for `CarthageFactory`

Reply:
Are you sure you have THE SAME addresses for this swap contracts on mainnet and testnet?

Reply:
Are you sure you have THE SAME addresses for these swap contracts on mainnet and testnet?

Reply:
Yes, they're deployed on candle chain

Reply:
> First letter:
> 
> `CarthageRouter` -> `carthageRouter`
> 
> 
> 
> The same for `CarthageFactory`

Fixed!